package.json

```diff
@@ -2,27 +2,24 @@
   "name": "ui-kit",
   "private": true,
   "scripts": {
-    "postinstall": "husky install & patch-package && npm run link:angular -w @coveo/atomic-angular-samples",
-    "start": "lerna run --parallel start",
-    "dev": "npm run start -- --scope @coveo/headless --scope @coveo/atomic",
-    "build": "lerna run clean && lerna run build",
-    "test": "lerna run --parallel test",
-    "test:watch": "lerna run --parallel test:watch --no-prefix",
-    "lint:check": "eslint .",
-    "lint:fix": "eslint --fix .",
+    "postinstall": "husky install && patch-package && npm run link:angular -w @coveo/atomic-angular-samples",
+    "dev:atomic": "concurrently \"npm run -w @coveo/headless dev\" \"npm run -w @coveo/atomic dev\"",
```

Comment:
Defining a dependency in Nx between dev scripts and parent dev scripts doesn't work currently, since it runs the parent's dev script (Headless in this case) and then hangs instead of starting the second one. This will have to be handled in a case-by-case basis.

Until then, I'm using `concurrently` to run both of these scripts in parallel.

Reply:
You cant use --target=targetA,targetB ?

Reply:
Maybe, but I know nx has a limit of parallel targets because they're meant to be processes that end eventually. We can change that limit (as far as I'm aware), but I feel like I'd be working against Nx's principles.

Until I improve the `dev` scripts so that they are "resolved" once the page is served, I prefer to keep it like this. It'll be replaced eventually anyway.

Reply:
Oh yeah, it's watch scripts, nevermind carry on.